core/services/vrf/listener_v2.go

```diff
@@ -492,7 +503,7 @@ func (lsn *listenerV2) processRequestsPerSubBatch(
 	start := time.Now()
 	var processed = make(map[string]struct{})
 	startBalanceNoReserveLink, err := MaybeSubtractReservedLink(
-		lsn.l, lsn.q, startBalance, lsn.chainID.Uint64(), subID)
+		lsn.q, startBalance, lsn.chainID.Uint64(), subID)
 	if err != nil {
 		lsn.l.Errorw("Couldn't get reserved LINK for subscription", "sub", reqs[0].req.SubId)
```

Comment:
I think we should include the `err` returned from `MaybeSubtractReservedLink` as a field in this log - `lsn.l.Errorw(..., "err", err)` since it doesn't look like it was included before (perhaps because we were logging in that method anyway?)

Reply:
Oh yeah good catch! didn't realize it wasn't being logged